services/f-droid/f-droid.service.js

```diff
@@ -1,19 +1,28 @@
 'use strict'
 
 const Joi = require('@hapi/joi')
+const {
+  optionalNonNegativeInteger,
+  nonNegativeInteger,
+} = require('../validators')
 const { addv } = require('../text-formatters')
 const { version: versionColor } = require('../color-formatters')
-const { BaseYamlService, InvalidResponse } = require('..')
+const { BaseJsonService, NotFound } = require('..')
 
 const schema = Joi.object({
-  CurrentVersion: Joi.alternatives().try(Joi.string(), Joi.number()).required(),
+  packageName: Joi.string().required(),
+  suggestedVersionCode: optionalNonNegativeInteger,
+  packages: Joi.array().items({
+    versionName: Joi.string().required(),
+    versionCode: nonNegativeInteger,
+  }),
 }).required()
 
 const queryParamSchema = Joi.object({
-  metadata_format: Joi.string().valid('yml', 'txt'),
+  suggested: Joi.equal(''),
```

Comment:
Is it worth documenting this parameter in the second `F-Droid (latest by suggested)` example?

For example, here's another badge that has some documentation: https://github.com/badges/shields/blob/master/services/discord/discord.service.js

Reply:
Instead of documenting this option, I decided to switch to `include_prereleases` instead of `suggested`. This option is negative to another, and it's more clear and already widely used in other badges.

Reply:
Indeed more clear, no documentation needed.